Source/core/resources/exchangeRates.json

```diff
@@ -0,0 +1,173 @@
+{  
```

Comment:
Yes, that occurred to me but then the data we have is a few years old. I was in fact thinking of finding historic exchange rates from some source to resolve this issue. As for your concern, I'll be looking for some free API that could provide us with real time exchange rates.

What do you think?